bazel/grpc_deps.bzl

```diff
@@ -519,3 +520,11 @@ def grpc_test_only_deps():
             ],
             build_file = "@com_github_grpc_grpc//third_party:constantly.BUILD",
         )
+
+    if "com_google_libprotobuf-mutator" not in native.existing_rules():
+        new_git_repository(
```

Comment:
please change the new_git_repository to http_archive -  http_archive is preferred and I don't see a reason why you wouldn't be able to use it  (since it's a github hosted repo).

Reply:
done... didn't realize it was a possibility!